RPG Wizardry/Assets/Scripts/GameWorld/CameraManager.cs

```diff
@@ -0,0 +1,41 @@
+﻿using nl.SWEG.RPGWizardry.Player;
+using nl.SWEG.RPGWizardry.Utils.Behaviours;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace nl.SWEG.RPGWizardry.GameWorld
+{
+    [RequireComponent(typeof(Camera))]
+    public class CameraManager : SingletonBehaviour<CameraManager>
+    {
+        /// <summary>
+        /// Transform of the player
+        /// </summary>
+        private Transform playerTransform;
+
+        #region Methods
+        /// <summary>
+        /// Gets reference to player transform
+        /// </summary>
+        // Start is called before the first frame update
+        private void Start()
+        {
+            if (PlayerManager.Exists)
+            {
+                playerTransform = PlayerManager.Instance.GetComponent<Transform>();
```

Comment:
Don't use GetComponent for Transform. Use .transform instead

Reply:
Fixed